docker/run.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+#set -e
+
+if [ ${PGID+x} ]; then
+  echo "Adding user netdata to group with id ${PGID}"
```

Comment:
Is this documented somewhere for users to know when/how to use this? If not, then we should.

Reply:
Probably the best place would be in [wiki page regarding docker installation]( https://github.com/firehol/netdata/wiki/Install-netdata-with-Docker). I am planing on simplifing that page and promote this installation method by adding links in [netdata installation page](https://github.com/firehol/netdata/wiki/Installation).

Probably will do it today or tomorrow.

Reply:
Awesome! sounds good.

Might be worth adding it in to the readme file too, since that is what gets shown on the dockerhub page automatically I believe.

Reply:
How is the "Full Description" on dockerhub managed? https://hub.docker.com/r/firehol/netdata/ Is this just the Readme of the source repo? It contains a log of generic information but no USAGE information for the docker images.
The dockerhub description and the Wiki page should also clearly show the limitations of this installation method.

Reply:
I hadn't tried it before, but just did a quick test here [0] and it seems that docker will read a README.md in the directory the dockerfile is in, then the root one only if that is not found. (source is at [1]).

So.... We should be able to add a readme for the docker image in the `/docker` folder

[0] https://hub.docker.com/r/justin8/dockertest/
[1] https://github.com/justin8/dockertest